Packs/CommonTypes/ReleaseNotes/3_2_3.md

```diff
@@ -0,0 +1,3 @@
+
+#### Indicator Types
+- **urlRep**: Fixed an issue where regex failed to parse url addresses.
```

Comment:
Let's elaborate on the fixed introduced here, like the support for ipv6 and the rest.